### PR TITLE
Add to the Earth Space Station Program

### PR DESCRIPTION
**Pull request by Teykn**
Adds a set of 3 contracts based on the Skylab station. 1st contract requires launching a more advanced station and crewing it with 3 crew for 30d. 2nd contract involves crewing for 60d, and 3rd contract involves crewing for 90d. Both 2nd & 3rd contract can be completed twice, with the former intended to be required and the latter optional.

Also allows `Duration` timers to count down while outside the vessel scene, allowing for players to focus on other contracts. (Should a note/popup be added to let players know they can focus on other contracts?)

### CODE REVIEW

**Comment by siimav** [comment]
It appears that the cfg file validator found some errors in your files.

**Comment by Teykn** [comment]
Fixed. Had a typo

**Comment by StonesmileGit** [comment]
Regarding a note about being able to leave the station while the timer ticks down:
If this is not the case for all contracts converted to Programs, then yes, some note in the description of the contract or in the parameter in-flight would be nice

**Comment by siimav** [comment]
Have you tested what happens if the crew capsule or the station itself gets destroyed? Does the contract get locked to vessels that no longer exist? If that does indeed happen then take a look at the new VPG parameters that were added for the X-plane contracts.

**Comment by Teykn** [comment]
I've tested it with the station but not the crew capsule. When station is deleted, current contract fails and you start over from the 1st contract. I haven't done that for the crew capsule, but that shouldn't be too hard to implement.

**Comment by siimav** [comment]
Some notes:
* Some of those contracts still have deadlines. This doesn't really make sense with programs and in fact I recently nuked those from all existing ones.
* In some cases disableOnStateChange is at the very end of VPG which is somewhat hard to read
* Noticed a couple of places where the indenting is off a bit

Otherwise if you have validated that these work then I imagine it's good to merge.

File: GameData/RP-0/Contracts/Space Stations/Improved Space Station.cfg [hunk 1]
```diff
@@ -0,0 +1,606 @@
...
+		program = EarthSpaceStation
+	}
+	REQUIREMENT
+	{
+		name = CompleteContract
+		type = CompleteContract
+		contractType = first_ISScrew
+		minCount = 2
+	}
+	RREQUIREMENT
```
Comment (Falki-git) [inline GameData/RP-0/Contracts/Space Stations/Improved Space Station.cfg:479]: typo

Comment (Falki-git) [reply GameData/RP-0/Contracts/Space Stations/Improved Space Station.cfg:479]: Also, in order for these contracts to show up, [this contract group](https://github.com/KSP-RO/RP-0/blob/NotBudgets/GameData/RP-0/Contracts/Groups.cfg#L124) needs to be renamed to SpaceStations

Comment (Teykn) [reply GameData/RP-0/Contracts/Space Stations/Improved Space Station.cfg:479]: Thanks for pointing out the typo, will get to fixing it.

Wasn’t the contract group and program renamed from SpaceStations to EarthSpaceStation, presumably with a Lunar, Martian, etc programs planned in the future? I noticed that and decided to go along with it.

Comment (Falki-git) [reply GameData/RP-0/Contracts/Space Stations/Improved Space Station.cfg:479]: Sure, but your changes in this PR renamed the group of all contracts back to SpaceStations. So either rename them back to EarthSpaceStation or rename the entire group to SpaceStations, preferably the former :)

File: GameData/RP-0/Contracts/Space Stations/Improved Space Station.cfg [hunk 2]
```diff
@@ -243,7 +243,7 @@ CONTRACT_TYPE
 {
 	name = first_ISScrew
 	title = Crew your Improved Space Station for 60 days and collect science
-	group = SpaceStations
+	group = EarthSpaceStation
 	agent = Stations
 	
 	description = You have succeeded in crewing your station for a month. Now it's time for you to crew it for longer. Crew your station for 60 days and then return safely. You'll be staying for longer intervals in the future. This mission can be completed 2 times. &br;<b><color=red>Do NOT deorbit the station while you are completing this contract. Otherwise, this contract will fail and you will be forced to redo the Improved Space Station contract</color></b> &br;&br;<b>Number of Contracts Completed: $first_ISScrew_Count</b>
```
Comment (Falki-git) [inline GameData/RP-0/Contracts/Space Stations/Improved Space Station.cfg:249]: Line 274. Rep reward of 25 for first ISS seems a bit low, no?

File: GameData/RP-0/Contracts/Space Stations/Improved Space Station.cfg [hunk 3]
```diff
@@ -447,7 +447,7 @@ CONTRACT_TYPE
 	maxExpiry = 0
 	maxCompletions = 2
 	maxSimultaneous = 1
-	deadline = 365 * RP1DeadlineMult() // 1 year
+	deadline = 0
 	
 	targetBody = HomeWorld()
```
Comment (Falki-git) [inline GameData/RP-0/Contracts/Space Stations/Improved Space Station.cfg:453]: Line 459. Rep reward here also seems low

**Comment by Teykn** [comment]
@Falki-git Regarding rep rewards, what would be reasonable values? I’m not good at balancing, so tried to set values that weren’t too high

**Comment by Falki-git** [comment]
Neither am I, but looking at other contracts that are at the same "tier", I'd maybe go with:
- first_spaceStation = 300
- improved_spaceStation = 200
- first_ISScrew = 200
- second_ISScrew = 300

For reference, these are some of other contracts currently:
- crewed first orbit = 200
- first EVA = 150
- orbit3 = 250
- lunar crewed flyby = 300 (commented out)
- lunar crewed orbit = 200 (commented out)
- first crewed landing = 400

**Comment by siimav** [comment]
Has anyone got the chance to test the contracts ingame?
I would need to merge this before I can pull in the changes from latest release into NotBudgets.

**Comment by Teykn** [comment]
> Has anyone got the chance to test the contracts ingame? I would need to merge this before I can pull in the changes from latest release into NotBudgets.

Tested it. Parameters involving the crew capsule are reset when it's destroyed, so working as intended. Parameter where the crew stay for X duration isn't reset if it's complete though. Should it be reset if the crew capsule is destroyed? If not, this can be merged.

**Pull request merged**